oneflow/user/kernels/dim_scatter_kernel_util.h

```diff
@@ -105,7 +105,8 @@ OF_DEVICE_FUNC void DoDimScatter(const DimOpIndexNdHelper<IDX_T>& src_nd_helper,
 #else
       std::cerr << "The index element " << idx_elem << " is out of bounds for dimension " << dim
                 << " with size " << upper_bound << std::endl;
-      throw Error::CheckFailedError();  // TODO: Remove throw Error.
+      UNIMPLEMENTED();
```

Comment:
```suggestion
      UNIMPLEMENTED() << 上面106-107 的报错信息;
```

直接把报错和抛异常合并吧，我记得说可以的。如果不可以就参考下其他地方怎么抛异常前打印报错信息的。